docs/quickstart.md

```diff
@@ -0,0 +1,760 @@
+    from aws_lambda_powertools.logging import correlation_paths
+
+    logger = Logger(service="APP")
+    tracer = Tracer()
+    app = ApiGatewayResolver()
+
+
+    @app.get("/hello/<name>")
+    @tracer.capture_method
+    def hello_name(name):
+        tracer.put_annotation("User", name)
+        logger.info(f"Request from {name} received")
+        return {"statusCode": 200, "body": {"message": f"hello {name}!"}}
+
+
+    @app.get("/hello")
+    @tracer.capture_method
+    def hello():
+        tracer.put_annotation("User", "unknown")
+        logger.info(f"Request from unknown received")
+        return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+
+    @logger.inject_lambda_context(correlation_id_path=correlation_paths.API_GATEWAY_REST, log_event=True)
+    @tracer.capture_lambda_handler
+    def lambda_handler(event, context):
+        return app.resolve(event, context)
+    ```
+    
+Thanks to using powertools tracer we have much cleaner code right now. 
+In order to make our methods visible in traces we added `@tracer.capture_method` decorator to processing methods. 
+We added  annotations directly in code without adding it with context manager using `tracer.put_annotation` method. 
+Since we added `@tracer.capture_lambda_handler` decorator for our `lambda_handler`, powertools will automatically add information about cold start as an annotation. 
+It also automatically adds lambda response as a metadata into trace so we don't need to worry about it as well.
+!!! tip 
+    To read about differences between annotation and metadata in traces please follow [link](https://awslabs.github.io/aws-lambda-powertools-python/latest/core/tracer/#annotations-metadata). 
+As a result, you should see traces for your lambda in the X-RAY console.
+=== "Example X-RAY Console View"
+![Tracer utility](./media/tracer_utility_showcase_2.png)
+
+You may also consider using **CloudWatch ServiceLens** which ties together CloudWatch metrics and logs, in addition to traces from AWS X-Ray.
+It gives you a complete view of your applications and their dependencies enhancing the observability of your services. 
+From there you can easily navigate to specific logs in CloudWatch Logs Insight, Metrics Dashboard or Traces in CloudWatch X-Ray traces.
+=== "Example CloudWatch ServiceLens View"
+![CloudWatch ServiceLens View](./media/tracer_utility_showcase_3.png)
+!!! Info
+    If you want to learn more about CloudWatch ServiceLens follow this [link](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/ServiceLens.html)
+## Custom Metrics
+Last step in order to provide full observability is to add some business level metrics. 
+Lambda adds low level technical metrics (Invocations, Duration, Error Count & Success Rate etc) into CloudWatch Metrics out of the box. 
+You have decided to add some custom metrics in order to increase visibility into what happens with your application. 
```

Comment:
Can be removed, since it repeats the information for the first sentence.

Reply:
done